src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryNumber = string.replace('$', '').replace(',', '');
```

Comment:
This line of code removes the '$' and ',' symbols from a string, but it will only remove the first occurrence of each. If you want to remove all occurrences, you need to use a regular expression with the 'g' flag. For example: string.replace(/\$/g, '').replace(/,/g, '').